.pre-commit-config.yaml

```diff
@@ -51,6 +51,7 @@ repos:
       - fastapi~=0.65.1
       - uvicorn[standard]>=0.12.0,<0.14.0
       - pydantic~=1.8.2
+      - graphene
```

Comment:
just checking that these thoughts no longer apply 
https://github.com/dev-zero/aiida-graphql#requirements

Reply:
what using strawberry instead? well the key thing there is that graphene is what is recommended for integration into fastapi: https://fastapi.tiangolo.com/advanced/graphql/.I don't know if that is possible with strawberry, and well so far I haven't had any issues with graphene.
In terms of popularity: 6.6K stars for https://github.com/graphql-python/graphene, and 1k stars of https://github.com/strawberry-graphql/strawberry, so you could say graphene is slightly the more used (I'm sure both a re fine).

Reply:
FYI this is issue tracking graphql v3 in graphene: https://github.com/graphql-python/graphene/issues/1127, which hopefully should be sooner than later, and doesn't look like it would break any of the current code: https://github.com/graphql-python/graphene/wiki/v3-release-notes